### PR TITLE
Some improvements to Chart documentation

### PR DESCRIPTION
**Pull request by denisgolius**
This PR contains: 
- adds some additional information about VM Cluster Chart
- adds some additional information about k8s-stack Chart

### CODE REVIEW

File: charts/victoria-metrics-cluster/README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ This chart will do the following:
 
 * Rollout victoria metrics cluster
 
+> Note that this chart only installs VictoriaMetrics cluster components such as vminsert, vmselect and vmstorage. It will not create or configure metrics scraping. If you are looking for a chart to configure monitoring stack in cluster check out [victoria-metrics-k8s-stack chart](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-k8s-stack#helm-chart-for-victoria-metrics-kubernetes-monitoring-stack).
```
Comment (hagen1778) [inline charts/victoria-metrics-cluster/README.md:19]: I think this is Chart Details. Lines 15-17 can be removed.

**Review by zekker6 — DISMISSED** [review]

File: charts/victoria-metrics-cluster/README.md [hunk 2]
```diff
@@ -12,9 +12,7 @@ Victoria Metrics Cluster version - high-performance, cost-effective and scalable
 
 # Chart Details
 
-This chart will do the following:
-
-* Rollout victoria metrics cluster
+Note that this chart only installs VictoriaMetrics cluster components such as vminsert, vmselect and vmstorage. It will not create or configure metrics scraping. If you are looking for a chart to configure monitoring stack in cluster check out [victoria-metrics-k8s-stack chart](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-k8s-stack#helm-chart-for-victoria-metrics-kubernetes-monitoring-stack).
```
Comment (tenmozes) [inline charts/victoria-metrics-cluster/README.md:15]: __Note__:  this chart installs VictoriaMetrics cluster components such as vminsert, vmselect and vmstorage. It doesn't create or configure metrics scraping. If you are looking for a chart to configure monitoring stack in cluster check out [victoria-metrics-k8s-stack chart](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-k8s-stack#helm-chart-for-victoria-metrics-kubernetes-monitoring-stack).

File: charts/victoria-metrics-cluster/README.md.gotmpl [hunk 3]
```diff
@@ -12,9 +12,7 @@
 
 # Chart Details
 
-This chart will do the following:
-
-* Rollout victoria metrics cluster
+Note that this chart only installs VictoriaMetrics cluster components such as vminsert, vmselect and vmstorage. It will not create or configure metrics scraping. If you are looking for a chart to configure monitoring stack in cluster check out [victoria-metrics-k8s-stack chart](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-k8s-stack#helm-chart-for-victoria-metrics-kubernetes-monitoring-stack).
```
Comment (tenmozes) [inline charts/victoria-metrics-cluster/README.md.gotmpl:15]: as above

**Review by tenmozes — CHANGES_REQUESTED** [review]

**Review by tenmozes — APPROVED** [review]

**Comment by tenmozes** [comment]
@denisgolius thank you for contribution

**Pull request merged**